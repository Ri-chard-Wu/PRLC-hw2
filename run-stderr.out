[pid 2][job()] pop failed.
[pid 3][job()] pop failed.
[pid 1][job()] pop failed.
[pid 1][job()] pop failed.
[pid 3][job()] pop failed.
[pid 3][job()] pop failed.
[pid 1][job()] pop failed.
[pid 3][job()] pop failed.
[pid 1][job()] pop failed.
[pid 1][job()] pop failed.
[pid 3][job()] pop failed.
[pid 3][job()] pop failed.
[pid 3][job()] pop failed.
[pid 2][job()] pop failed.
[pid 2][job()] pop failed.
[pid 1][job()] pop failed.
[proc 0][init_procCtrlTbl()] after init, proc njob:
262144, 262144, 262144, 262144, 
[dynamic_job_schedule()] maxNJob: 256144, maxNJobPid: 1, minNJob: 246144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 524287 - 519288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 524287 - 519288
[dynamic_job_schedule()] maxNJob: 247499, maxNJobPid: 0, minNJob: 237144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 262143 - 257144
[dynamic_job_schedule()] maxNJob: 233144, maxNJobPid: 1, minNJob: 222144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 519287 - 514288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 519287 - 514288
[dynamic_job_schedule()] maxNJob: 230031, maxNJobPid: 0, minNJob: 219144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 257143 - 252144
[dynamic_job_schedule()] maxNJob: 226144, maxNJobPid: 2, minNJob: 216144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 786431 - 781432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 786431 - 781432
[dynamic_job_schedule()] maxNJob: 220144, maxNJobPid: 1, minNJob: 209144, minNJobPid: 3
[proc 1][dynamic_job_cancel()] recvSz: 5000, 514287 - 509288
[proc 3][dynamic_job_receive()] recvSz: 5000, 514287 - 509288
[dynamic_job_schedule()] maxNJob: 217144, maxNJobPid: 2, minNJob: 206144, minNJobPid: 3
[proc 2][dynamic_job_cancel()] recvSz: 5000, 781431 - 776432
[proc 3][dynamic_job_receive()] recvSz: 5000, 781431 - 776432
[dynamic_job_schedule()] maxNJob: 212589, maxNJobPid: 0, minNJob: 201144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 252143 - 247144
[dynamic_job_schedule()] maxNJob: 209144, maxNJobPid: 1, minNJob: 198144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 509287 - 504288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 509287 - 504288
[dynamic_job_schedule()] maxNJob: 208144, maxNJobPid: 2, minNJob: 197144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 776431 - 771432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 776431 - 771432
[dynamic_job_schedule()] maxNJob: 203491, maxNJobPid: 0, minNJob: 192144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 247143 - 242144
[dynamic_job_schedule()] maxNJob: 202144, maxNJobPid: 1, minNJob: 191144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 504287 - 499288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 504287 - 499288
[dynamic_job_schedule()] maxNJob: 199144, maxNJobPid: 2, minNJob: 188144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 771431 - 766432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 771431 - 766432
[dynamic_job_schedule()] maxNJob: 197144, maxNJobPid: 1, minNJob: 187144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 499287 - 494288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 499287 - 494288
[dynamic_job_schedule()] maxNJob: 196323, maxNJobPid: 0, minNJob: 186144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 242143 - 237144
[dynamic_job_schedule()] maxNJob: 194144, maxNJobPid: 2, minNJob: 183144, minNJobPid: 3
[proc 2][dynamic_job_cancel()] recvSz: 5000, 766431 - 761432
[proc 3][dynamic_job_receive()] recvSz: 5000, 766431 - 761432
[dynamic_job_schedule()] maxNJob: 190144, maxNJobPid: 1, minNJob: 180144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 494287 - 489288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 494287 - 489288
[dynamic_job_schedule()] maxNJob: 189180, maxNJobPid: 0, minNJob: 179144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 237143 - 232144
[dynamic_job_schedule()] maxNJob: 185144, maxNJobPid: 2, minNJob: 174144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 761431 - 756432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 761431 - 756432
[dynamic_job_schedule()] maxNJob: 183144, maxNJobPid: 1, minNJob: 173144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 489287 - 484288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 489287 - 484288
[dynamic_job_schedule()] maxNJob: 180144, maxNJobPid: 2, minNJob: 170144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 756431 - 751432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 756431 - 751432
[dynamic_job_schedule()] maxNJob: 180024, maxNJobPid: 0, minNJob: 169144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 232143 - 227144
[dynamic_job_schedule()] maxNJob: 176144, maxNJobPid: 1, minNJob: 166144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 484287 - 479288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 484287 - 479288
[dynamic_job_schedule()] maxNJob: 173144, maxNJobPid: 2, minNJob: 163144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 751431 - 746432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 751431 - 746432
[dynamic_job_schedule()] maxNJob: 173023, maxNJobPid: 0, minNJob: 162144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 227143 - 222144
[dynamic_job_schedule()] maxNJob: 169144, maxNJobPid: 1, minNJob: 159144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 479287 - 474288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 479287 - 474288
[dynamic_job_schedule()] maxNJob: 166144, maxNJobPid: 2, minNJob: 156144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 746431 - 741432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 746431 - 741432
[dynamic_job_schedule()] maxNJob: 166022, maxNJobPid: 0, minNJob: 155144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 222143 - 217144
[dynamic_job_schedule()] maxNJob: 164144, maxNJobPid: 1, minNJob: 154144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 474287 - 469288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 474287 - 469288
[dynamic_job_schedule()] maxNJob: 159144, maxNJobPid: 2, minNJob: 149144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 741431 - 736432
[proc 2][dynamic_job_cancel()] recvSz: 5000, 741431 - 736432
[dynamic_job_schedule()] maxNJob: 134347, maxNJobPid: 0, minNJob: 124144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 217143 - 212144
[dynamic_job_schedule()] maxNJob: 117144, maxNJobPid: 1, minNJob: 107144, minNJobPid: 3
[proc 3][dynamic_job_receive()] recvSz: 5000, 469287 - 464288
[proc 1][dynamic_job_cancel()] recvSz: 5000, 469287 - 464288
[dynamic_job_schedule()] maxNJob: 76144, maxNJobPid: 3, minNJob: 66144, minNJobPid: 2
[proc 2][dynamic_job_receive()] recvSz: 5000, 464288 - 469287
[proc 3][dynamic_job_cancel()] recvSz: 5000, 464288 - 469287
[dynamic_job_schedule()] maxNJob: 57243, maxNJobPid: 0, minNJob: 47144, minNJobPid: 2
[proc 2][dynamic_job_receive()] recvSz: 5000, 212143 - 207144
[dynamic_job_schedule()] maxNJob: 48144, maxNJobPid: 1, minNJob: 38144, minNJobPid: 2
[proc 1][dynamic_job_cancel()] recvSz: 5000, 464287 - 459288
[proc 2][dynamic_job_receive()] recvSz: 5000, 464287 - 459288

[check_receive_terminate_signal()][proc 1]             receive terminate signal. Terminating...


[check_receive_terminate_signal()][proc 2]             receive terminate signal. Terminating...


[check_receive_terminate_signal()][proc 3]             receive terminate signal. Terminating...

dt: 41764865 us
