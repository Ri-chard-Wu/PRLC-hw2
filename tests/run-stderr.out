srun: Force Terminated job 4293773
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** STEP 4293773.0 ON apollo32 CANCELLED AT 2023-03-17T18:52:28 DUE TO TIME LIMIT ***
srun: error: apollo32: task 0: Terminated
